dunai-frp-bearriver/src/FRP/BearRiver.hs

```diff
@@ -617,3 +617,109 @@ replaceOnce a = dSwitch (arr $ const (a, Event ())) (const $ arr id)
 
 -- ** Tuples
 dup  x     = (x,x)
+
+-- * Monad transformations
```

Comment:
Can you put this section in a separate file? @ivanperez-keera maybe has a good idea for a name.

Reply:
Done. I just called it `FRP.BearRiver.Monad`.